test/version_table_test_case.py

```diff
@@ -30,7 +30,6 @@ def setUp(self):
         self.ddb_client.create_table(TableName=config.dynamo_object_version_table_name,
                                      AttributeDefinitions=[
                                          dict(AttributeName=VersionService.key_name, AttributeType='S'),
-                                         dict(AttributeName=VersionService.value_name, AttributeType='S')
```

Comment:
I realize that this was inconsistent with the table definition in TF config, but why do we not declare non-key attribute in TF?

cc: @noah-aviel-dove

Reply:
I have no idea. I wrote the TF template over a year ago.

Reply:
it looks like `non_key_attributes` are only available for a `global_secondary_index` https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table

From inspecting the object_versions table for my local deployment it does not look like we use a `global_secondary_index`.

Reply:
> I have no idea. I wrote the TF template over a year ago.

That's a lame response, @noah-aviel-dove. If you wrote this code, you are accountable. I also forget things about code I wrote, but if I do, I dive in and come back with an answer.